src/wallet/db.cpp

```diff
@@ -167,8 +167,8 @@ bool CDB::Recover(const std::string& filename, void *callbackDataIn, bool (*reco
     // Call Salvage with fAggressive=true to
     // get as much data as possible.
     // Rewrite salvaged data to fresh wallet file
-    // Set -rescan so any missing transactions will be
-    // found.
+    // call rescanblockchain (RPC) so any missing
```

Comment:
Yes. You right. The rescan is not part of the call. But the part that calls CDB::Recover does always call a rescan. But now I got your point. We should refer to `ScanForWalletTransactions()` instead to `rescanblockchain (RPC)`. Right?